src/parsing/parsers/generic/jison/insert/insert.test.json

```diff
@@ -204,5 +204,50 @@
         ]
       }
     }
+  },
+  {
+    "namePrefix": "should suggest tables without filled table columns",
+    "beforeCursor": "INSERT INTO ",
+    "afterCursor": " ()",
+    "expectedResult": {
+      "lowerCase": false,
+      "suggestTables": {},
+      "suggestKeywords": ["TABLE"],
+      "suggestDatabases": {
+        "appendDot": true
+      }
+    }
+  },
+  {
+    "namePrefix": "should suggest tables without filled table columns and values",
+    "beforeCursor": "INSERT INTO ",
+    "afterCursor": " () VALUES ()",
+    "expectedResult": {
+      "lowerCase": false,
+      "suggestTables": {},
+      "suggestKeywords": ["TABLE"],
+      "suggestDatabases": {
+        "appendDot": true
+      }
+    }
+  },
+  {
+    "namePrefix": "should suggest columns with filled table and empty values",
+    "beforeCursor": "INSERT INTO some_table (",
+    "afterCursor": ") VALUES ()",
+    "expectedResult": {
+      "lowerCase": false,
+      "suggestColumns": {
+        "tables": [
+          {
+            "identifierChain": [
+              {
+                "name": "some_table"
+              }
+            ]
+          }
+        ]
+      }
+    }
   }
```

Comment:
We also need a test case for the next query `INSERT INTO table() VALUES(|)` (`|` is a cursor), right?

Reply:
I'm not pretty sure, because if our cursor inside of the VALUES brackets anyway we will have empty context, with errors and without them, because we don't suggest here anything

Reply:
My bad, you are correct.